src/main/java/net/io/kino/model/Show.java

```diff
@@ -0,0 +1,17 @@
+package net.io.kino.model;
+
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+@JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "id")
+public class Show {
```

Comment:
Encje powinny być nazywane zgodnie z terminologią ustaloną, żeby nikomu się nie myliło. Tutaj to chyba ShowTime?